apps/ensadmin/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "ensadmin",
+  "private": true,
+  "version": "0.0.1",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "vite build",
+    "preview": "vite preview",
+    "test": "vitest",
+    "lint": "eslint . && biome check --write .",
```

Comment:
ESLint was pre-configured with Vite.js setup, so I've just composed biome lint into the `lint` command.

@shrugs what do you think about that?

Reply:
eslint can/should be fully replaced by biome, but you can use the biome migrate feature to clone vite's preferred eslint config and use that as a biome.json in this subdirectory. with biome we don't need eslint installed anywhere

https://biomejs.dev/guides/migrate-eslint-prettier/

Reply:
although it might not be comprehensive enough, vite does seem to have a bunch of eslint stuff going on... in that case i'm not sure what the best approach is, but would suggest not using biome at all, and instead enforcing eslint-only in that subdir. adding that subdirectory to our biome's `ignore` field, for example.

Reply:
Thanks for that suggestion, will apply it tomorrow 🚀